WebContent/WEB-INF/jsp/collection.jsp

```diff
@@ -43,9 +44,29 @@
 	    var collIndex = document.mainForm.category.selectedIndex;
 		var parentIndex = document.mainForm.parentCollection.selectedIndex;  
 	    if(parentIndex != 0){
-	    	if (category.options[collIndex].value == parentCollection.options[parentIndex].value) {
-	    		alert("Parent collection can't be the same collection. Please select another parent collection.");
+	    	var collTypeValue = document.mainForm.collType.options[collTypeIndex].value;
+	    	var parentColl = document.mainForm.parentCollection.options[parentIndex].text;
+	    	var parentCollType = parentColl.substring(parentColl.lastIndexOf("[")+1, parentColl.lastIndexOf("]"));
+
+			if (collTypeValue == collTypes[2]) { // ProvenanceCollectionPart
+	    		alert("A " + collTypes[2] + " is not allowed to have any parent collections!");
 				return false;
+	    	} else {
+	    		var error = "";
+	    		if (category.options[collIndex].value == parentCollection.options[parentIndex].value) {
+		    		error = "Parent collection can't be the same collection.";
+		   		} else if (collTypeValue == collTypes[1]) { // ProvenanceCollection
+		   			if (parentCollType != collTypes[2])
+	    				error = "A " + collTypes[1] + " can only have a " + collTypes[2] + " parent.";
```

Comment:
Yes, you are right, Matt. I realize that something is messing up when I refactoring the if conditional blocks and I've corrected it:
https://github.com/ucsdlib/damsmanager/commit/903578f6de170881b0d006c6fc197ea7ad59f480